sauce/features/accounts/customFlagNames/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'CustomFlagNames',
+  type: 'checkbox',
+  default: false,
+  section: 'accounts',
+  title: 'Adds Custom Flag Names',
+  description: 'Adds the ability to change set custom flag names. **WARNING** Flag names are stored in localStorage and will be lost if localStorage is cleared.'
```

Comment:
You might also consider something less drastic like "Please note!" or just "Note!" 

You also have a double verb at the begging of the sentence. "...ability to change set custom..."